### DISCUSSION TITLE
Github Action Implementation.

### DISCUSSION THREAD

**Discussion by CharlesMSilva** [mobile-dev-inc/Maestro#284] [Q&A] [answered]
Hello! 

Would that work with github action? If so, how would I do?

**Comment by axelniklasson** [comment, marked as answer]
Hi there @CharlesMSilva! You can use the `action-maestro-cloud` GitHub action to upload your app and Flows to Maestro Cloud. More information is available [here](https://cloud.mobile.dev/ci-integration/github-actions).
